ksp_plugin_adapter/map_node_pool.cs

```diff
@@ -306,16 +306,25 @@ public class SingleProvenancePool {
         }
         case MapObject.ObjectType.PatchTransition: {
           CelestialBody celestial = properties.reference_frame.Centre();
-          // TODO(phl): If we have a non-BS time, give it here.
+          double speed = properties.velocity.magnitude;
           caption.Header = L10N.CacheFormat("#Principia_MapNode_ImpactHeader",
                                             source,
                                             celestial.Name());
```

Comment:
Let’s show the altitude as well, like we do for the apsides (ll. 266–272 above).

Reply:
![impact](https://github.com/mockingbirdnest/Principia/assets/7423020/315b4b54-9301-4194-b2a4-5a697ae209c0)

Done.